package/MDAnalysis/analysis/rms.py

```diff
@@ -265,34 +267,25 @@ def process_selection(select):
     return select
 
 
-class RMSD(object):
-    """Class to perform RMSD analysis on a trajectory.
-
-    Run the analysis with :meth:`RMSD.run`, which stores the results
-    in the array :attr:`RMSD.rmsd`::
```

Comment:
The git diff didn't handle tracking the changes super well. I believe if you look at the file what you're asking for was never deleted from the class string. Can you check?